### ISSUE TITLE
Access to the script /public has been denied

### ISSUE THREAD

**Issue by NathMorgan** [phalcon/box#60]
## Steps to reproduce

Upgraded the box and Phalcon Vagrant box script

### Expected behavior

View the webpage

### Actual behavior

'Access denied.' in the browser and in the logs:

> 2017/04/23 10:04:14 [error] 2440#2440: *1 FastCGI sent in stderr: "Access to the script '/home/vagrant/workspace/scritch.es/public' has been denied (see security.limit_extensions)" while reading response header from upstream, client: 192.168.0.11, server: scritch.es.dev, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.1-fpm.sock:", host: "scritch.es.dev"

### Host operating system

Manjaro Linux

### `settings.yml`
```yaml
---
name: box
ip: "192.168.50.4"
memory: 2048
cpus: 2

authorize: ~/.ssh/id_rsa.pub

folders:
    - map: /home/tuqiri/Documents/development/scritch.me
      to: /home/vagrant/workspace/scritch.es

sites:
    - map: scritch.es.dev
      to:  /home/vagrant/workspace/scritch.es/public

databases:
    - scritch.es
```
### Vagrant up output

https://gist.github.com/NathMorgan/3f69581c7d574580ffe440794e1fa469

### Details

- Vagrant: 1.9.2
- Provider: Virtualbox 5.1.18
- Phalcon Box: v2.0.0-beta4
- Operating System: Manjaro Linux

**Comment by NathMorgan** [comment]
Fixed the issue by changing cgi.fix_pathinfo=0 to cgi.fix_pathinfo=1 in the php.ini file

**Comment by sergeyklay** [comment]
Fixed in the `v2.0.0`. Thank you
